include/barrett/bus/abstract/communications_bus.h

```diff
@@ -45,7 +45,7 @@ namespace bus {
 class CommunicationsBus {
 public:
 	static const size_t MAX_MESSAGE_LEN = 8;  /** The maximum of any of the available communications buses */
-	static const double TIMEOUT = 1.0;  /** Bus connection timeout limit in seconds */
+	static const double TIMEOUT = 0.01;  /** Bus connection timeout limit in seconds */
```

Comment:
10 ms makes sense. In that case, the comment "*bus connection timeout* limit in seconds" is misleading. Can you change that to be accurate?